components/nodemanager-service/pgdb/nodes.go

```diff
@@ -958,10 +962,12 @@ func (db *DB) UpdateNodeDetectInfo(ctx context.Context, nodeDetectInfo *nodes.No
 	if err != nil {
 		return errors.Wrap(err, "UpdateNodeDetectInfo unable to read rows affected")
 	}
-	if rows == 1 {
-		logrus.Debugf("updated node %s", nodeDetectInfo.NodeId)
+	if rows == 0 {
```

Comment:
i was seeing the "unexpected results" msg a lot. seems like that's actually because 0 nodes were being updated but we weren't explicitly checking for that case